src/modules/access-wallet/ModuleAccessWalletSoftware.vue

```diff
@@ -0,0 +1,258 @@
+          :handler-access-wallet="accessHandler"
+          @unlock="unclockWallet"
+        />
+        <!--
+        =====================================================================================
+         Access With PrivateKey
+        =====================================================================================
+        -->
+        <access-wallet-private-key
+          v-else-if="walletType === types[2]"
+          :handler-access-wallet="accessHandler"
+          @unlock="unclockWallet"
+        />
+        <!--
+        =====================================================================================
+          Warning
+        =====================================================================================
+        -->
+        <mew-warning-sheet
+          title="Not Recommended"
+          description="This information is sensetive, and these options should only be used in offline settings by experienced crypto users."
+          :link-obj="warningSheetObj"
+          class="mt-6"
+        />
+      </v-sheet>
+
+      <!--
+        <access-mnemonic
+          v-else-if="showMnemonic"
+          :btn-call="btnCall"
+          :unlock-mnemonic-wallet="unlockMnemonicWallet"
+          :step="step"
+          :set-mnemonic-path="setMnemonicPath"
+          :hw-wallet-instance="hwWalletInstance"
+          :set-address="setAddress"
+        />
+      </v-sheet> -->
+      <div class="spacer-y-medium" />
+    </template>
+  </mew-overlay>
+</template>
+
+<script>
+import mewSuperButton from '@/components/mewSuperButton/MewSuperButton';
+import AccessWalletKeystore from './software/components/AccessWalletKeystore';
+import accessMnemonic from './software/components/AccessMnemonic';
+import AccessWalletPrivateKey from './software/components/AccessWalletPrivateKey';
+import { mapActions, mapState } from 'vuex';
+import { Toast, ERROR } from '@/modules/toast/handler/handlerToast';
+import { SOFTWARE_WALLET_TYPES } from './software/handlers/helpers';
+import hadlerAccessWalletSoftware from './software/handlers/hadlerAccessWalletSoftware';
+
+export default {
+  name: 'ModuleAccessWalletSoftware',
+  components: {
+    AccessWalletKeystore,
+    accessMnemonic,
+    AccessWalletPrivateKey,
+    mewSuperButton
+  },
+  props: {
+    open: {
+      type: Boolean,
+      default: false
+    },
+    close: {
+      type: Function,
+      default: () => {}
+    },
+    walletType: {
+      type: String,
+      default: ''
+    }
+  },
+  data() {
+    return {
+      types: SOFTWARE_WALLET_TYPES,
+      warningSheetObj: {
+        title: 'Learn More',
+        url: 'https://kb.myetherwallet.com/en/offline/using-mew-offline'
+      },
+      buttons: [
+        /* Keystore Button */
+        {
+          label: 'Keystore',
+          description: 'Access via Keystore',
+          icon: require('@/assets/images/icons/icon-keystore-file.svg'),
+          fn: () => {
+            this.setType(SOFTWARE_WALLET_TYPES[0]);
```

Comment:
lets switch this to Object so itll be `SOFTWARE_WALLET_TYPES.KEYSTORE`

Reply:
fixed in https://github.com/MyEtherWallet/MyEtherWallet/pull/2846/commits/2f678616ff35a0e9dccfbdce859132cf5e069d58